Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -29,6 +29,8 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
     fileprivate var _pinchZoomEnabled = false
     fileprivate var _doubleTapToZoomEnabled = true
     fileprivate var _dragEnabled = true
+    fileprivate var _dragYEnabled = true
```

Comment:
Shall we change to `false` by default? Ideally this should be set by user explicitly

Reply:
`dragEnabled` is true by default. Having `dragYEnabled` false by default would change the current default behaviour.

Reply:
oops, my bad. Lots of `||` and `!`..